src/test/java/com/puppycrawl/tools/checkstyle/bdd/InlineConfigParser.java

```diff
@@ -86,4 +87,35 @@ private static void setCheckName(InputConfiguration.Builder inputConfigBuilder,
         final String checkPath = getPackageFromFilePath(filePath) + "." + checkName + "Check";
         inputConfigBuilder.setCheckName(checkPath);
     }
+
+    private static void setCheckProperties(InputConfiguration.Builder inputConfigBuilder,
+                                      String filePath, List<String> lines)
+                    throws CheckstyleException {
+        int lineNo = 2;
+        for (String line = lines.get(lineNo); !line.contains("*/"); line = lines.get(++lineNo)) {
+            if (line.isEmpty()) {
+                continue;
+            }
+            if (line.contains("(default)")) {
+                try {
+                    final int equalPosition = line.indexOf('=');
+                    if (line.contains("(default)")) {
+                        final int defaultEndPosition = line.indexOf(')');
+                        final String property = line.substring(0, equalPosition - 1);
```

Comment:
`final String property = line.substring(0, equalPosition - 1);` and `inputConfigBuilder.addDefaultProperty(property, value);` are common for two subconditions, you can put these statements before and after `if`

Reply:
one is ```addDefaultProperty``` and the other is ```addNonDefaultProperty```. Can't take it out.
I have done the other one.